delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/syntax/IOResourceFSyntax.scala

```diff
@@ -0,0 +1,18 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.syntax
+
+import ch.epfl.bluebrain.nexus.delta.sdk.model.ResourceF
+import monix.bio.IO
+
+trait IOResourceFSyntax {
```

Comment:
Since we have the opportunity let's do something more generic:
```scala
trait IOFunctorSyntax {
  implicit final def ioFunctorOps[E, A: Functor](io:IO[E, A]): IOFunctorOps = new IOFunctorOps(io)
}
final class IOFunctorOps[E, A: Functor](private val io:IO[E, A]) {
   def mapValue[B](f: A => B): IO[E, B] = io.map(_.map(f))
}
```